### DISCUSSION TITLE
Getting unable to connect to Github SVN repository

### DISCUSSION THREAD

**Discussion by jitendra-koodo** [community/community#86620] [Repositories] [answered]
### Select Topic Area

General

### Body

I have a SVN repo in Github. I use SVN client to access this repo. Its a 4 years old repo, working fine till now. Now Getting connection error :

1. Checked from Bangalore, India location, Windows machine, TortoiseSVN client : 
Error: Unable to connect to a repository at URL , path not found 
(Repo exists as I can see it on github.com)

1. Checked from Mumabi, India location, Linux machine, command line client : same error 
Error: Unable to connect to a repository at URL , path not found 

This means its not network issue or client or proxy issue at my end. Someone else facing same issue? This was working till day before yesterday.Suddenly something has changed.

**Comment by spenserblack** [comment, marked as answer]
[According to GitHub's changelog](https://github.blog/changelog/2024-01-08-subversion-has-been-sunset/), they have dropped support for Subversion (SVN).

**Reply by EjayCinchez** [reply]
I also got this error... so sad...

**Reply by jitendra-koodo** [reply]
There was no communication in advance before final shutdown.. did not receive any email

**Reply by MarcellStar1** [reply]
I also experienced something similar. After seeing this post, at least I know the cause of the problem

**Comment by diraneyya** [comment]
For the workflows that used to rely on SVN for downloading a part of a repository (e.g. download a specific folder, without cloning and hence without history information), this is a possible solution:
- https://gist.github.com/diraneyya/c2171c2a262ac53f0d296c84d9d72e5c
- https://medium.com/@orwa.diraneyya/github-drops-svn-support-and-breaks-legacy-software-2a30bedbd08d
